.prettierignore

```diff
@@ -28,6 +28,7 @@ change
 __snapshots__
 
 # package specific files
+packages/make-styles-webpack-loader/__fixtures__/error-syntax/code.js
```

Comment:
wont turning off prettier inline via pragma in that file be enough? using globals for these kind of scenarios is not a good idea/pattern

Reply:
Do you mean `// prettier-ignore`? If so, it does not work:

```
packages\make-styles-webpack-loader\__fixtures__\error-syntax\code.js: SyntaxError: Unexpected token, expected "," (8:1)
[error]   6 |   root: { color: 'red' },
[error]   7 | }
[error] > 8 | 
[error]     | ^
error Command failed with exit code 2.
```

Reply:
to bad, thx for veryfing !

Reply:
another idea would be to make these kind of invalid files as a `.txt`, and modify the extension as needed in testing scenarios. WDYT?

Reply:
Makes sense, 0370ea9.

Reply:
nice